### PR TITLE
All workers in /include now moved to /src/worker

### PR DESCRIPTION
**Pull request by annando**
We are now done with removing all workers from the old place. Once, the two worker calls in the addons are changed as well, we can remove the old code from the worker calls.

### CODE REVIEW

**Comment by MrPetovan** [comment]
Related to #3878

**Comment by MrPetovan** [comment]
Shoot, I was working on those file to move `include/Contact.php` to `src/` 😅

I'm expecting a big ol' merge conflict.

**Comment by annando** [comment]
I was the first to make the PR 😛

**Comment by MrPetovan** [comment]
I know it's first past the post!

File: src/Worker/CreateShadowentry.php [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+ * @brief This script creates posts with UID = 0 for a given public post.
+ *
+ * This script is started from mod/item.php to save some time when doing a post.
+ */
+
+namespace Friendica\Worker;
+
+require_once("include/threads.php");
+
+class CreateShadowentry {
```
Comment (MrPetovan) [inline src/Worker/CreateShadowentry.php]: Can you please rename it to `CreateShadowEntry` please?

Comment (annando) [reply src/Worker/CreateShadowentry.php]: I guess this is some German compound wording :-)

**Pull request merged**